specification/conformance/conformance_rules/columns/commitmentdiscountquantity.json

```
@@ -0,0 +1,512 @@
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "CommitmentDiscountQuantity-C-011-C"
+                    },
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "CommitmentDiscountQuantity-C-012-C"
+                    }
+                ]
+            },
+            "Condition": {
+                "CheckFunction": "CheckValue",
+                "ColumnName": "ChargeCategory",
+                "Value": null
+            },
+            "Dependencies": [
+                "CommitmentDiscountQuantity-C-011-C",
+                "CommitmentDiscountQuantity-C-012-C"
+            ]
+        }
+    },
+    "CommitmentDiscountQuantity-C-011-C": {
+        "Function": "Validation",
+        "Reference": "CommitmentDiscountQuantity",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Dynamic",
+        "ValidationCriteria": {
+            "MustSatisfy": "CommitmentDiscountQuantity MUST be the quantity of CommitmentDiscountUnit, paid fully or partially upfront, that is eligible for consumption over the commitment discount's term when ChargeFrequency is 'One-Time'.",
+            "Keyword": "MUST",
+            "Requirement": {},
+            "Condition": {
+                "CheckFunction": "CheckValue",
+                "ColumnName": "ChargeFrequency",
+                "Value": "One-Time"
+            },
+            "Dependencies": []
+        }
+    },
+    "CommitmentDiscountQuantity-C-012-C": {
+        "Function": "Validation",
+        "Reference": "CommitmentDiscountQuantity",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Dynamic",
+        "ValidationCriteria": {
+            "MustSatisfy": "CommitmentDiscountQuantity MUST be the quantity of CommitmentDiscountUnit that is eligible for consumption for each charge period that corresponds with the purchase when ChargeFrequency is 'Recurring'.",
+            "Keyword": "MUST",
+            "Requirement": {},
+            "Condition": {
+                "CheckFunction": "CheckValue",
+                "ColumnName": "ChargeFrequency",
+                "Value": "Recurring"
+            },
+            "Dependencies": []
+        }
+    },
+    "CommitmentDiscountQuantity-C-013-C": {
+        "Function": "Composite",
+        "Reference": "CommitmentDiscountQuantity",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Static",
+        "ValidationCriteria": {
+            "MustSatisfy": "When ChargeCategory is 'Usage' adheres to the following additional requirements:",
```

*see [this](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/1209/files#r2415263056) comment*

```suggestion
            "MustSatisfy": "When ChargeCategory is \"Usage\", CommitmentDiscountQuantity adheres to the following additional requirements:",
```